_posts/2024-12-12-socratic-seminar-142.md

```diff
@@ -0,0 +1,395 @@
+---
+layout: post
+type: socratic
+title: "Bitcoin Developers’ Socratic Seminar 142"
+meetup: "https://www.meetup.com/bitdevsnyc/events/304565259/"
+---
+
+## Announcements
+Please join us for our next Socratic Seminar. A special thank you to our sponsors [Chaincode Labs](https://chaincode.com) and [Wolf NYC](https://wolfnyc.com) for food, refreshments and event space.
+
+If you can't make it to the main event please join us at [PUBKEY](https://pubkey.bar/home) around 9:30PM.
+
+Follow New York BitDevs on [Twitter](https://x.com/BitDevsNYC)
+
+Follow us on NOSTR: `npub1nydevscv0slx9eyr0chkkdgzqfhtp4guk039akhq6g9fnsrf6lcsu`
+
+## Presentation
+- Lava loans DLC-based protocol
+
+## Mailing Lists, Meetings and Bitcoin Optech
+### Mailing Lists
+#### [bitcoin-dev](https://groups.google.com/g/bitcoindev)
+- [Adding New BIP Editors](https://groups.google.com/g/bitcoindev/c/cuMZ77KEQAA)
+- [Difficulty in emulating "weaker" OP_SUCCESS and why it should be a real opcode](https://groups.google.com/g/bitcoindev/c/UgCUykOyd7s)
+- [Covenants Support - Bitcoin Wiki](https://groups.google.com/g/bitcoindev/c/fdxkE1Al4TI)
+- [Great Consensus Cleanup Revival](https://groups.google.com/g/bitcoindev/c/CAfm7D5ppjo)
```

Comment:
One annoying thing you have to do manually is go through all the ML posts and remove anything that was originally started before the target time period. The scraper lists every thread **either created or replied-to** in the provided window, due to the limitation of the google API 🤷

Reply:
I took care of this and pushed